javascript/javascript3/week2/currency-caculator.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" type="text/css" href="currency-caculator.css" />
+		<title>Warmup Exercises</title>
+	</head>
+	<body>
+		<div class="form-style-3">
+			<fieldset>
+				<legend>Currency Converter</legend>
+				<label for="field1"
+					><span>Amount </span
+					><input type="number" class="input-field" id="amount"
+				/></label>
+
+				<label for="field4"
```

Comment:
Instead of giving field4, try something that gives insight into what you are planning to do

Reply:
yes sure i will change